processor/resourcedetectionprocessor/internal/system/system.go

```diff
@@ -69,6 +71,17 @@ func NewDetector(p processor.CreateSettings, dcfg internal.DetectorConfig) (inte
 	}, nil
 }
 
+// toIEEERA converts a MAC address to IEEE RA format.
+// Per the spec: "MAC Addresses MUST be represented in IEEE RA hexadecimal form: as hyphen-separated
+// octets in uppercase hexadecimal form from most to least significant."
+// Golang returns MAC addresses as colon-separated octets in lowercase hexadecimal form from most
```

Comment:
Just as a future reference for myself or others, I was wondering where this was documented/guaranteed, and found it in the implementation of [String()](https://pkg.go.dev/net#HardwareAddr.String). 👍